CMakeLists.txt

```diff
@@ -88,10 +88,25 @@ macro(jerry_add_compile_flags)
   jerry_add_flags(CMAKE_C_FLAGS ${ARGV})
 endmacro()
 
+# SHOULD THIS BE, SAY, AT THE TOP OF THE FILE?
+if(CMAKE_C_COMPILER_ID MATCHES "GNU")
```

Comment:
Yes, it's correct in both cases, I was wondering how to avoid this `USING_GCC` variable, since the CMAKE_COMPILER_IS_GNUCC is also true when CMAKE_C_COMPILER_ID is
"GNU",  but I think you're right and using directly the CMAKE_C_COMPILER_ID to determine the GNU compiler is closely aligned with the conventional way.